src/network/http.py

```diff
@@ -1,15 +1,25 @@
+"""
```

Comment:
The whole file is obsolete, this was used for testing asyncore and proxies, can be removed.